src/js/menu/menu.js

```diff
@@ -54,6 +65,72 @@ class Menu extends Component {
 
     return el;
   }
+
+  /**
+   * Handle key press for menu
+   *
+   * @param {Object} event Event object
+   * @method handleKeyPress
+   */
+  handleKeyPress (event) {
+    if (event.which === 37 || event.which === 40) { // Left and Down Arrows
+      event.preventDefault();
+      this.stepForward();
+    } else if (event.which === 38 || event.which === 39) { // Up and Right Arrows
```

Comment:
I've been thinking of grabbing this lib: https://github.com/timoxley/keycode
Seems to be small and tested and does exactly what we want.